tests/network/vmi_networking.go

```diff
@@ -250,6 +250,7 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 			// Use alpine because cirros dhcp client starts prematurely before link is ready
 			masqIface := libvmi.InterfaceDeviceWithMasqueradeBinding()
 			masqIface.Model = "e1000"
+			masqIface.PciAddress = "0000:02:01.0"
```

Comment:
Commit message: Could you please elaborate why is there a need to enhance the vendor assertion?

Reply:
Thank you for adding more information.